### ISSUE TITLE
How to implement weight decay towards the pre-trained model?

### ISSUE THREAD

**Issue by sedol1339** [modelscope/ms-swift#2189]
Hello, let me one question.

If using SWIFT for supervised fune-tuning, how do I implement penalizing the distance between starting and current weights? This was shown to be effective in https://arxiv.org/abs/1706.03610

**Comment by tastelikefeet** [comment]
https://github.com/modelscope/ms-swift/blob/main/swift/trainers/mixin.py#L588 
change the code here please
